src/unix/mod.rs

```diff
@@ -321,9 +321,19 @@ pub const ATF_PUBL: ::c_int = 0x08;
 pub const ATF_USETRAILERS: ::c_int = 0x10;
 
 pub const FNM_PERIOD: c_int = 1 << 2;
-pub const FNM_CASEFOLD: c_int = 1 << 4;
 pub const FNM_NOMATCH: c_int = 1;
 
+cfg_if! {
+    if #[cfg(any(
+        target_os = "illumos",
+        target_os = "solaris",
+    ))] {
+        pub const FNM_CASEFOLD: c_int = 0x08;
```

Comment:
```rust
        pub const FNM_CASEFOLD: c_int = 1 << 3;
```

To be consistent, unless the headers say differently (got a link by the way?)

Reply:
Illumos (Solaris) reference: https://github.com/illumos/illumos-gate/blob/aaceae985c2e78cadef76bf0b7b50ed887ccb3a6/usr/src/head/fnmatch.h#L41

Reply:
Thanks! Added to the PR description so it gets in history.